server/services/courtRegisterService.test.ts

```diff
@@ -436,16 +436,14 @@ describe('Court Register service', () => {
 
       expect(updatedCourtBuilding).toEqual(courtBuilding)
     })
-    it('will send null rather than blanks', async () => {
+    it('will send no attribute rather than blanks', async () => {
       const buildingWithBlanks: UpdateCourtBuilding = { ...courtBuilding, locality: '', subCode: '  ' }
       await courtRegisterService.updateCourtBuilding({ username: 'tommy' }, 'SHFCC', '1', buildingWithBlanks)
 
       expect(updatedCourtBuilding).toEqual({
         buildingName: 'Crown Square',
         street: 'High Street',
-        locality: null,
         town: 'Sheffield',
-        subCode: null,
         postcode: 'S1 2BJ',
         county: 'South Yorkshire',
```

Comment:
small functional change above - the JSON node is now absent since using `undefined` which is preferred